addon/webextension/background/senderror.js

```diff
@@ -81,6 +81,16 @@ this.senderror = (function() {
     });
   };
 
+  // Every copy of Screenshots has its own unique moz-extension:// URL.
+  // Raven doesn't correctly group errors with different URLs, so we use the
+  // raven.js `dataCallback` option to turn the UUID into "screenshots-addon"
+  // before sending.
+  // TODO: do this here or somewhere else, though? like in reportError?
+  function normalizeURL(data) {
+    data.request.url = data.request.url.replace(/moz-extension:\/\/[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}/, "resource://screenshots-addon");
```

Comment:
If we were fancy we could actually rewrite it to the github URL, but maybe later. Mostly we need to jam the github revision in somewhere in the add-on, though I guess `https://raw.githubusercontent.com/mozilla-services/screenshots/{version}/addon/webextension/{file}` would work for our releases. I believe Sentry gives you nice links if you give it fetchable URLs.